docs/classes.rst

```diff
@@ -367,7 +367,9 @@ Attempting to bind ``Pet::set`` will cause an error since the compiler does not
 know which method the user intended to select. We can disambiguate by casting
 them to function pointers. Binding multiple functions to the same Python name
 automatically creates a chain of function overloads that will be tried in
-sequence.
+the order they were defined. If the ``py::prepend()`` tag is added to the
+definition, the function will be placed at the begining of the overload sequence
```

Comment:
If the definition is marked with `py::prepend()`, ...

?

Reply:
Same remark about "marked" as above.

Reply:
This text covers basic usage of pybind11 for people who learn how to create bindings for classes for the first time. This new `py::prepend` tag scratches an itch that will only arise in very rare/advanced usage of pybind11, hence I don't think this is a good place to talk about it. The most sensible place to put it would be in an (IIRC currently non-existent) overview of all of the tags that can be specified in a function binding declaration.

Reply:
I've moved it to the place in advanced where the sequence is discussed, is that better?